src/site/content/en/blog/media-session/index.md

```diff
@@ -390,6 +400,79 @@ Resetting the position state is as easy as setting it to `null`.
 navigator.mediaSession.setPositionState(null);
 ```
 
+## Video conferencing actions {: #video-conferencing-actions }
+
+When the user puts their video call into a Picture-in-Picture window, the
+browser may display controls for the microphone, camera, and for hanging up.
```

Comment:
```suggestion
browser may display controls for the microphone and camera, and for hanging up.
```

Reply:
@jpmedley  As you can see in the screenshot below, hanging up is also a control (icon in the PiP window)

![image](https://user-images.githubusercontent.com/634478/117350073-69f1e300-aeac-11eb-9035-7cc7f90d92e4.png)

Reply:
Done